omnibus/files/private-chef-cookbooks/private-chef/recipes/default.rb

```diff
@@ -52,6 +52,17 @@
   node.set['private_chef']['bootstrap']['enable'] = false
 end
 
+# Do a sanity check to make sure both SAML and LDAP not enabled at the same time
+chef_manage = { 'chef_manage' => {} }
```

Comment:
This bit (56-60) might make more sense in the config recipe around here: https://github.com/chef/chef-server/blob/master/omnibus/files/private-chef-cookbooks/private-chef/recipes/config.rb#L39